models/models.js

```diff
@@ -0,0 +1,21 @@
+const db =require('../db/connection.js')
+
+
+const selectTopics = () => {
+    return db
+        .query('SELECT * FROM topics;').then((result) => {
+    
+            return result.rows
+    })
+}
+
+const selectArticleById = (article_id) => {
+    return db
+        .query('SELECT * FROM articles WHERE article_id = $1;', [article_id])
+        .then(result => {
+    
```

Comment:
Have a look at my comments in your controller and have a go at implementing it 🙂